csrankings-c.csv

```diff
@@ -375,7 +375,7 @@ Chao Xu 0006,Peking University,http://www.cis.pku.edu.cn/faculty/vision/xuchao/x
 Chao Yang 0015,Hunan University,http://csee.hnu.edu.cn/people/yangchao,NOSCHOLARPAGE
 Chao Yang 0016,Xidian University,https://faculty.xidian.edu.cn/yc,Z8_q6TcAAAAJ
 Chao Zhang 0001,Peking University,http://www.cis.pku.edu.cn/faculty/vision/zhangchao/zhangchao.htm,i-vJ_-UAAAAJ
-Chao Zhang 0008,Tsinghua University,https://www.cs.tsinghua.edu.cn/publish/csen/4623/2020/20200105192656183949129/20200105192656183949129_.html,NOSCHOLARPAGE
+Chao Zhang 0008,Tsinghua University,https://www.cs.tsinghua.edu.cn/info/1127/3590.htm,YreXlgsAAAAJ

```

Comment:
English translation of page says "Zhang Chao (part-time on campus)". Please explain.